packages/gutenpride-template/package.json

```
@@ -0,0 +1,20 @@
+{
+	"name": "@wordpress/gutenpride-template",
+	"version": "1.0.0-prerelease",
+	"description": "A package containing a block template to use with @wordpress/create-block.",
```

It would be nice to make it clear that this is an official template from the tutorial.
```suggestion
	"description": "Template for @wordpress/create-block used in the official WordPress tutorial.",
```

---

Done!